datalad/tests/utils.py

```diff
@@ -12,100 +12,71 @@
 import glob
 import gzip
 import inspect
+import logging
 import lzma
-import shutil
-import stat
-from json import dumps
-import os
-import re
-import tempfile
-import platform
 import multiprocessing
 import multiprocessing.queues
-import logging
+import os
+import platform
 import random
+import re
+import shutil
 import socket
+import ssl
+import stat
+import tempfile
 import textwrap
-import warnings
-from fnmatch import fnmatch
 import time
-from difflib import unified_diff
+import warnings
 from contextlib import contextmanager
-from unittest.mock import patch
-import ssl
-
+from difflib import unified_diff
+from fnmatch import fnmatch
+from functools import wraps
 from http.server import (
     HTTPServer,
     SimpleHTTPRequestHandler,
 )
-
-from functools import wraps
+from json import dumps
 from os.path import (
     curdir,
     exists,
-    join as opj,
-    relpath,
-    split as pathsplit,
-)
-
-from nose.plugins.attrib import attr
-from nose.tools import (
-    assert_equal,
-    assert_false,
-    assert_greater,
-    assert_greater_equal,
-    assert_in as in_,
-    assert_in,
-    assert_is,
-    assert_is_none,
-    assert_is_not,
-    assert_is_not_none,
-    assert_not_equal,
-    assert_not_in,
-    assert_not_is_instance,
-    assert_raises,
-    assert_true,
-    eq_,
-    make_decorator,
-    ok_,
-    raises,
 )
+from os.path import join as opj
+from os.path import relpath
+from os.path import split as pathsplit
+from unittest.mock import patch
 
-from nose.tools import assert_set_equal
-from nose.tools import assert_is_instance
+import pytest
 from nose import SkipTest
```

Comment:
couldn't we avoid this `import`, or when would we be able to get rid of it? (the idea is to remove `nose` as a dependency altogether)

Reply:
That import is unnecessary, and was presumably re-introduced during some rebase.  I've removed it again.